config/chime_science_run_recv.yaml

```diff
@@ -191,11 +194,18 @@ count_check:
     kotekan_stage: countCheck
     in_buf: visbuf_5s_26m_gated
 
+delay_frames:
+  num_frames_to_hold: 3072
```

Comment:
You need to set `visbuf_10s_flags` to have `> 3072` frames for this config to work.  It might be worth asking Richard to check if this how he wanted to delay things here.

Reply:
Ah yes, it needs to be a bit bigger than 30s. @jrs65 how many frames should we hold here? 3072 frames is 30 seconds, so do we want 4096 frames or 3584?

Reply:
I'm trying to figure that one out now. I need to know how much memory we are going to put in there first. Will get back to this soon.